src/workflows/bidsSpatialPrepro.m

```diff
@@ -1,34 +1,48 @@
 % (C) Copyright 2019 CPP BIDS SPM-pipeline developers
 
 function bidsSpatialPrepro(opt)
-  % bidsSpatialPrepro(opt)
   %
   % Performs spatial preprocessing of the functional and structural data.
   %
-  % TODO update description
-  % The structural data are segmented and normalized to MNI space.
-  % The functional data are re-aligned, coregistered with the structural and
-  % normalized to MNI space.
+  % USAGE::
+  %
+  %   bidsSpatialPrepro([opt])
+  %
+  % :param opt: structure or json filename containing the options. See
+  %             ``checkOptions()`` and ``loadAndCheckOptions()``.
+  % :type opt: structure
+  %
+  % The anatomical data are segmented, skulls-stripped [and normalized to MNI space].
+  %
+  % The functional data are re-aligned (unwarped), coregistered with the structural,
+  % the anatomical data is skull-stripped [and normalized to MNI space].
+  %
+  % If you do not want to:
+  %
+  % - to perform realign AND unwarp, make sure you set
+  %   ``opt.realign.useUnwarp`` to ``false``.
```

Comment:
you meant "true" to set in order to run realign & unwrap?

Reply:
Oops !!! Can you open a PR to fix that too ?! :-)